### DISCUSSION TITLE
Add support for self hosting the data files?

### DISCUSSION THREAD

**Discussion by lancejpollard** [danigb/smplr#68] [Ideas]
Any chance you are planning on making it so this great library doesn't have to rely on third-party HTTP domains to fetch data? Perhaps you could make it pluggable, just add a `setConfigUrl('/base/path')` sort of thing, and then allow placing the files underneath that spot to self host. Something along those lines.

Just a thought!

**Comment by danigb** [comment]
Yes, something like this should definitely be possible. Do you have a specific use case?
